hat_parts/common_functions.c

```diff
@@ -94,6 +94,9 @@ string hatlog_dir() {
 
 string hatlog_file() { return hatlog_dir() + "hat.log"; }
 
+string hatlog_sense_file() { return "/w/"+
```

Comment:
/d/Qc/ wont have write permissions to the author's dirs, so it will need to write to somewhere in /d/Qc/, I'd suggest /d/Qc/hat/log/<wizname>/senses.log

In hat_parts/common_functions.c there is already a setup_hatlog() to create the appropriate dir and with an ACCESS file, and a hatlog_dir() function to get the directory, and finally hat_log() to perform the actual logging.  We could adjust hat_log() to accept a 2nd parameter for the file name and just use that existing function for any logging operations. In this case we'd need to adjust the one existing call to hat_log() in out_line() to also use that 2nd parameter.